customization.md

```diff
@@ -0,0 +1,200 @@
+# DRAFT: Customizing CIF Core Components
```

Comment:
Is this still a draft or shall we also review that?

Reply:
This can be reviewed as well. I added `DRAFT` to indicate that this file will not be merged, but moved into the wiki.